main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+)
+
+func createPhoneBook(names []string, numbers []string) map[string]string {
+	m := make(map[string]string)
+	for i := range names {
+		m[names[i]] = numbers[i]
+	}
+	return m
+}
+func main() {
+	f, err := os.Open("phone_numbers.txt")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+
+	s := bufio.NewScanner(f)
+	var names, numbers []string
+	i := 0
+	for s.Scan() {
+		if i == 0 {
+			names = append(names, s.Text())
+			i = 1
+		} else {
+			numbers = append(numbers, s.Text())
+			i = 0
+		}
+	}
```

Comment:
This can also be replaced with 

```
for s.Scan() {
        names = append(names, s.Text())
        s.Scan()
        numbers = append(numbers, s.Text())
}
```